tutorials/CIFAR_test.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.utils.data.dataloader as dataloader
+import torch.optim as optim
+
+from torch.autograd import Variable
+from torchvision import transforms
+from torchvision.datasets import CIFAR10
+
+from resnet import ResNet18
+from metal import EndModel
+from metal.utils import convert_labels
+from torch.utils.data import Dataset
+
+import argparse
+
+
+SEED = 1
+
+# CUDA?
+cuda = torch.cuda.is_available()
+
+# For reproducibility
+torch.manual_seed(SEED)
+
+if cuda:
+    torch.cuda.manual_seed(SEED)
+    
+
+parser = argparse.ArgumentParser(description='PyTorch Training')
+
+parser.add_argument('--epochs', default=10, type=int,
+                    help='number of total epochs to run')
+
+parser.add_argument('--start-epoch', default=0, type=int,
+                    help='manual epoch number (useful on restarts)')
+parser.add_argument('-b', '--batch-size', default=10, type=int,
+                    help='mini-batch size (default: 1)')
+
+parser.add_argument('--lr', '--learning-rate', default=0.1, type=float,
+                    help='initial learning rate')
+
+parser.add_argument('--momentum', default=0.9, type=float, help='momentum')
+parser.add_argument('--weight-decay', '--wd', default=0, type=float,
+                    help='weight decay (default: 1e-4)')
+parser.add_argument('--print-freq', '-p', default=10, type=int,
+                    help='print frequency (default: 10)')
+    
+
+# The following identity module is to essentially replace the last FC layer
+# in the resnet model by the FC in MeTal
+
+class IdentityModule(nn.Module):
+    """A default identity input module that simply passes the input through."""
+
+    def __init__(self):
+        super().__init__()
+
+    def reset_parameters(self):
+        pass
+
+    def forward(self, x):
+        return x
+
+# Here we create a dataloader that transforms CIFAR labels from 0-9, to 1-10,
+# We do this because MeTal treats a 0 label as abstain
+class MetalDataset(Dataset):
```

Comment:
Since we already have a `MetalDataset`, I would rename this `OneIndexedDataset`?  Also clean up commented out stuff

Reply:
And this can go in `metal.utils`